environment/config.go

```diff
@@ -51,6 +55,34 @@ type Configuration struct {
 
 	// StaticConfiguration is used for running M3DB with a static config
 	Static *StaticConfiguration `yaml:"static"`
+
+	// Presence of a (etcd) server in this config denotes an embedded cluster
+	EmbeddedServer *EmbeddedKV `yaml:"embeddedServer"`
```

Comment:
Could we call this `seedNodeServer`?  It more accurately describes what we're doing, we should name everything around the nodes that run etcd as "seed nodes".